src/legacy/Routes.tsx

```diff
@@ -553,6 +554,11 @@ class RouteApp extends React.Component<any, any> {
               render={() => <WalletScreen info={info} />}
             />
 
+            <Route
```

Comment:
I don't think we need this in the legacy. So can we remove it?

Reply:
Removed